internal/service/secretsmanager/secret_rotation.go

```diff
@@ -38,6 +38,11 @@ func ResourceSecretRotation() *schema.Resource {
 				Type:     schema.TypeBool,
 				Computed: true,
 			},
+			"rotate_immediately": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
The markdown doc says this should default to false. I believe this is causing all newly made secrets to try to rotate immediately, potentially causing issues or unexpected results.

Reply:
Hi @MagikHands, this should be an error in documentation.
The default behavior should be set to true, just as the [SDK](https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.49.13/service/secretsmanager#RotateSecretInput).

The original issue mentioned to have the possibility to disable this default behavior.
[Github issue](https://github.com/hashicorp/terraform-provider-aws/issues/34961)

If you agree with this please open a new issue to update documentation.

Reply:
@MagikHands, I have create an issue to report this : https://github.com/hashicorp/terraform-provider-aws/issues/35702